api/tests.py

```diff
@@ -196,3 +197,52 @@ def test_bad_delete_request_returns_404(self):
         response = c.delete(f'/api/v1/users/{user.id}/scheduled_activities/5')
 
         self.assertEqual(404, response.status_code)
+
+
+class ScheduledActivityManagementCommandTest(TestCase):
+    def test_script_commmand(self):
+        activity = Activity.objects.create(name="Kayaking")
+
+        user = User.objects.create(username="test_user", first_name="Test", last_name="Name", email="test@example.com")
+
+        scheduled_activity_1 = ScheduledActivity.objects.create(
+            date="2020-04-14",
+            location="Denver, CO",
+            forecast="Sunny",
+            forecast_img="sunny",
+            temperature=45.20,
+            temp_hi=60.00,
+            temp_low=23.00,
+            precip_probability=0.07,
+            activity=activity,
+            user=user
+            )
+
+        scheduled_activity_2 = ScheduledActivity.objects.create(
+            date="2020-04-19",
+            location="Fort Collins, CO",
+            forecast="Meteorite Shower",
+            forecast_img="meteorite_shower",
+            temperature=450.20,
+            temp_hi=600.00,
+            temp_low=230.00,
+            precip_probability=0.07,
+            activity=activity,
+            user=user
+            )
+
+        call_command('script')
+
+        activities_after = ScheduledActivity.objects.all().order_by('updated_at')
```

Comment:
I like the approach to testing this here. I wouldn't have been sure of the best way to go about it but your approach makes sense. Nice job.

Reply:
Thanks took some time to see what worked